src/interface/css.py

```diff
@@ -0,0 +1,30 @@
+import streamlit as st
+
+
+def load_css():
+    st.markdown("""
+    <style>
+        :has(.st-key-plh1, .st-key-plh2) {
+            height: 100%;
+            overflow-y: auto;
+            padding-bottom: 0px;
+        }
+        .st-key-plh1, .st-key-plh2 {
+            overflow-x: hidden;
+        }
+          .st-emotion-cache-13ln4jf {
```

Comment:
I don't see the reason of adding this and another extra css. 
It is not elegant, and seems to be not errorproof. I would remove it.